lib/db.js

```diff
@@ -42,52 +42,58 @@ function get(collection, id) {
     });
 }
 
-
 function insertOne(collection, obj) {
     return new Promise((resolve, reject) => {
         db.collection(collection).insertOne(obj, (err, result) => {
         if (err) {
             console.log('err', err);
             return reject(err);
         }
-        console.log(`Inserted a document into collection ${collection}`);
         return resolve(result);
         });
     });
 }
 
 function updateOne(collection, obj) {
   return new Promise((resolve, reject) => {
-    var query = { name: obj.name };
-    delete obj['_id'];
+    var query = {'_id': ObjectID(obj._id)};
```

Comment:
:+1:

Reply:
once I saw the errors and more detail it made a lot more sense